drivers/storage/portworx/portworx.go

```diff
@@ -430,6 +430,10 @@ func (p *portworx) SetDefaultsOnStorageCluster(toUpdate *corev1.StorageCluster)
 
 	if preflight.IsEKS() {
 		toUpdate.Annotations[pxutil.AnnotationIsEKS] = "true"
+	} else if preflight.IsGKE() {
+		toUpdate.Annotations[pxutil.AnnotationIsGKE] = "true"
+	} else if preflight.IsAzure() {
+		toUpdate.Annotations[pxutil.AnnotationIsAKS] = "true"
```

Comment:
Azure does not mean AKS. It could be openshift on Azure or just vanilla k8s deployment.

Reply:
Ok then will add a way to just add the the provider if it exists.